Update Telenor_EPM_Collector.md (#908)
SkylineCommunications/dataminer-docs-connectors: connector/doc/Telenor_EPM_Collector.md

### Version Info

| Range                | Key Features                                         | Based on     | System Impact                                       |
|----------------------|------------------------------------------------------|--------------|-----------------------------------------------------|
| 1.0.0.x [Obsolete]   | Initial version                                      | -            | -                                                   |

> | 1.0.0.x [SLC Main]   | - Removed MAM Data IP Connection and associated logic. <br> - Channels Provisioning Filename is now configurable.  | 1.0.0.42     | Removed deprecated connection and associated logic. |

Reviewer: I'm not sure if the system impact is clear enough here: what does it mean for the user that the connection and associated logic have been deprecated? Will they have to recreate the element?

> [!NOTE]
> Starting from range 1.0.1.x, the channel file name must be configured manually. To do this, enter the desired value in the **Provisioning Channel File Name** parameter.

### Channel Details

The channel details are used to make the mapping of a channel. This is for Live DVB-C, Live IPTV, OTT Live/Start-Over and OTT Catchup. You can load these by clicking the **Load** button. The details will be loaded from the configured LPI folder in the sublocation *\enrichment\services\channels.csv*. However, note that **starting from range 1.0.1.x, the file name needs to be configured manually** (see [General](#general)).

The content of the loaded file must be comma-separated and must contain the following headers:
*ChannelID,ChannelName,CS_Type,DVB-C-Live-TV,IPTV-Live-TV,OTT-Live-TV,Start-over,Catchup,MultiCast,MC_Port,SID,TS_ID,ONid,MPEG-DASH OTT Live/Start-Over URL,HLS OTT Live/Start-Over URL,MPEG-DASH OTT Catch-Up URL,HLS OTT Catch-Up URL*

> [!NOTE]

> > Starting from range 1.0.0.X, another header is required: 'NPVR'

Author: It was a typo from my side :)

> *ChannelID,ChannelName,CS_Type,DVB-C-Live-TV,IPTV-Live-TV,OTT-Live-TV,Start-over,Catchup,MultiCast,MC_Port,SID,TS_ID,ONid,MPEG-DASH OTT Live/Start-Over URL,HLS OTT Live/Start-Over URL,MPEG-DASH OTT Catch-Up URL,HLS OTT Catch-Up URL*
> 
> > [!NOTE]
> > Starting from range 1.0.0.X, another header is required: 'NPVR'

Reviewer: Can we adjust this like this then?
```suggestion
*ChannelID,ChannelName,CS_Type,DVB-C-Live-TV,IPTV-Live-TV,OTT-Live-TV,Start-over,Catchup,MultiCast,MC_Port,SID,TS_ID,ONid,MPEG-DASH OTT Live/Start-Over URL,HLS OTT Live/Start-Over URL,MPEG-DASH OTT Catch-Up URL,HLS OTT Catch-Up URL,NVPR*

> [!NOTE]
> The "NPVR" header is only used from range 1.0.1.x onwards.
```
